acquia/acquia_cms
# ACMS-000: use acquia_cms_common module for logo and favicon path

**Motivation**
use acquia_cms_common module for logo and favicon path

**Proposed changes**
use acquia_cms_common module for logo and favicon path

**Alternatives considered**
NA

**Testing steps**
New install:
- install site with this branch
- logo and favicon file should be used from acquia_cms_common
- verify same by visiting /admin/appearance/settings
Existing install:
- update acquia_cms profile to this branch
- run database update
- logo and favicon file should be used from acquia_cms_common
- verify same by visiting /admin/appearance/settings

**Merge requirements**
